res_users_password_security/__openerp__.py

```diff
@@ -5,7 +5,7 @@
 
     'name': 'Password Security',
     "summary": "Allow admin to set password security requirements.",
-    'version': '9.0.1.0.1',
+    'version': '9.0.1.0.2',
```

Comment:
My point is follow your internal logic as this is not important at which number you start in the OCA repo. What is important though is that after the merge the numbers are dealt logically